start-proxy.ps1

```diff
@@ -1 +1 @@
-mitmproxy --allow-hosts api.teamwood.games -s mitmproxy-plugin.py
+mitmproxy --allow-hosts teamwood.games -s mitmproxy-plugin.py
```

Comment:
Has the domain also changed for the non-beta version?

Reply:
It's the same, but this domain prefix is able to capture both staging.teamwood.games and api.teamwood.games